Makefile

```diff
@@ -386,14 +386,12 @@ watcher: ## Install watcher operator via olm
 
 .PHONY: watcher_deploy
 watcher_deploy: ## Deploy watcher service
-	oc apply -f config/samples/watcher_requirements.yaml
```

Comment:
so we no longer need to create the secret ourselves, right? (I am unsure because I thought we were already using the 'telemetry created secret' and didn't realise we were creating this metric-storage-prometheus-config, or, was it just for the tests where we use the config/samples stuff?)

Reply:
In the previous patch, we had agreed the info and format with the telemetry team but we created it for the kuttl tests and the sample/tests because it was still in development in telemetry-operator. Now the patch which introduced it it's merged and in the operator in the openstack-operator-index.